### ISSUE TITLE
Swap in a logical partition

### ISSUE THREAD

**Issue by Petrouchka** [linuxmint/ubiquity#25]
After installing Linux Mint 17.3 bêta, I saw that swap partition is in a logical partition, alone in an extended partition.
Putting the swap partition at the end of the disk is a good idea because it's easiest to move the swap partition after cloning by example. But why not only create a primary partition for swap, instead of a logical partition ? It would be perfect.

**Comment by Cos-mo** [comment]
As the number of partitions in a MBR drive is limited to 4, placing swap into a primary partition might lead into trouble, if the user wants to create a new partition at a later time. Even resizing the existing partitions would not help. The user would have to delete swap, create the extended partition, create - besides the new partition he wants to create - a new swap and has to edit his fstab file, as the UUID of swap is now different to the original.

Extended partitions are a workaround against the limitation of the MBR partitioning system, but it does not hurt.

**Comment by Petrouchka** [comment]
No, using an extended partition is not a good idea because it impossible to move it with Gparted after cloning Linux Mint on bigger disks with Clonezilla. Using a primary partition is more easy to move.
If somebody want to use more than 4 partition (not a lot of people in normal users), they they will not use the swap partition that is too small.
